### PR TITLE
Housing counselor: Add responsive styling for search results header

### PR DESCRIPTION
**Pull request by anselmbradford**
Fixes [GHE]/CFGOV/platform/issues/3190

## Changes

- Remove `float:right` at mobile screen sizes for search result header and add left margin to links.

## Testing

1. `./frontend.sh`
2. Load find-a-housing counselor page and search for a ZIP code.
3. Scroll down to results header and resize down to mobile size.
4. See that results header text and links stack at small sizes.

### Before:

screen shot 2018-12-11 at 9 16 23 am

---

### After:

screen shot 2018-12-11 at 9 13 20 am

---

screen shot 2018-12-11 at 9 13 35 am

---

screen shot 2018-12-11 at 9 13 45 am

---

screen shot 2018-12-11 at 9 14 01 am

---

screen shot 2018-12-11 at 9 14 08 am

---

screen shot 2018-12-11 at 9 14 26 am

### CODE REVIEW

**Comment by schbetsy** [comment]
Is it possible that this change also added a scroll bar on the Paperwork Reduction Act text? Like so:

screen shot 2018-12-11 at 10 39 20 am

**Comment by schbetsy** [comment]
👆 Disregard that comment. I'm seeing the scroll bar on the PRA section on `master` too.

File: cfgov/unprocessed/apps/find-a-housing-counselor/css/hud.less [hunk 1]
```diff
@@ -15,7 +15,13 @@
 }
 
 .hud_hca_api_results_actions {
-    float: right;
+    .respond-to-min( @bp-xs-max, {
+        & {
+            float: right;
+            // Provide a gap between result header links and result text.
+            margin-left: 20px;
```
Comment (Scotchester) [inline cfgov/unprocessed/apps/find-a-housing-counselor/css/hud.less]: If I want to nitpick, `margin` should come before `float`, but I don't care that much.

Comment (anselmbradford) [reply cfgov/unprocessed/apps/find-a-housing-counselor/css/hud.less]: Sure, updated!

**Pull request merged**